README.md

```diff
@@ -1,20 +1,26 @@
-# kafka-connect-bigquery
+# Kafka-BigQuery Connector
```

Comment:
We'd like to stick with the Confluent idiom of "kafka-connect-foo" for directory/project names and "Kafka Connect <Foo> Connector" for referring to the connector itself (see [kafka-connect-hdfs](https://github.com/confluentinc/kafka-connect-hdfs) and [kafka-connect-jdbc](https://github.com/confluentinc/kafka-connect-jdbc)). Please change to "Kafka Connect BigQuery Connector".